extensions/PhabBugz/lib/Util.pm

```diff
@@ -305,4 +306,135 @@ sub set_phab_user {
   return Bugzilla->set_user($user, scope_guard => 1);
 }
 
+sub set_reviewer_rotation {
+  my ($revision) = @_;
+  my $rev_identifier = 'D' . $revision->id;
+
+  INFO("$rev_identifier: Setting reviewer rotation.");
+
+  # Load a fresh version of the revision with Heralds changes.
+  $revision = Bugzilla::Extension::PhabBugz::Revision->new_from_query(
+    {phids => [$revision->phid]});
+
+  # 1. Find out what the project reviewers and individual reviewers are.
+  # If the revision has a blocking reviewer group set, normally is 1) blocking
+  # and 2) ends in "-reviewers". Normally Herald will set this if certain
+  # conditions are met. If a blocking reviewer group cannot be found then
+  # do nothing.
+  my $blocking_project;
+  my @blocking_users;
+
+  foreach my $reviewer (@{$revision->reviews}) {
+    next if !$reviewer->{is_blocking};    # Only interested in blocking
+    if ($reviewer->{is_project}) {
+      next if $reviewer->{user}->name !~ /-reviewers$/;    # Only interested in reviewer groups
```

Comment:
From the spec it looks like the intention was for only review groups with `-reviewers-rotation` to be considered.

```suggestion
      # Only interested in opt-in reviewer rotation groups.
      next if $reviewer->{user}->name !~ /-reviewers-rotation$/;
```